hr_timesheet_predefined_description/models/account_analytic_line.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Tecnativa - Juan José Seguí
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = "account.analytic.line"
+
+    predefined_description_id = fields.Many2one(
+        comodel_name="timesheet.predefined.description", string="Predefined Description"
+    )
+
+    def write(self, vals):
```

Comment:
Always document the overriden method for expressing the intention of the code.

```suggestion
    def write(self, vals):
        # Fill the predefined description if no description provided
```